src/common/qengine.cl

```diff
@@ -47,6 +47,10 @@ inline real1 arg(const cmplx cmp)
                                                                                      \
     cmplx2 mulRes
 
+#define PREP_2X2_NORM()                                                              \
+    real1 norm_thresh = cmplxPtr[9];                                                 \
+    real1 dotMulRes
```

Comment:
Include the ; at the end here.

Reply:
Since the above has been consistent with the overall pattern, but it hurts nothing to follow your suggestion, I'll do so for every case of this pattern.